PSSwagger.LiveTestFramework/src/PSSwagger.LTF.Lib/Converters/LiveTestRequestConverter.cs

```diff
@@ -181,5 +188,13 @@ private bool ConvertObject(object val, Type expectedType, JsonSerializer seriali
 
             return success;
         }
+
+        private void Log(string msg, params object[] args)
+        {
+            if (this.module.Logger != null)
+            {
+                this.module.Logger.Log(msg, args);
```

Comment:
>msg [](start = 39, length = 3)

msg can be null or whitespace, right? Consider adding a check on msg too.

Reply:
e750658